src/en/authors-charm-metadata.md

```diff
@@ -3,22 +3,27 @@ Title: Charm metadata
 # Charm metadata
 
 The only file that must be present in a charm is `metadata.yaml`, in the root
-directory. A metadata file must be a valid yaml dictionary, containing at least
-the following fields:
+directory. It must be a valid YAML dictionary.
 
-  - `name` is the charm name, which is used to form the charm URL.
-    - It can only contain `a-z`, `0-9`, and `-`; must start with `a-z`; must not
-      end with a `-`; and may only end with digits if the digits are _not_
-      directly preceded by a space. Stick with names like `foo` and `foo-bar-baz`
-      and you needn't pay further attention to the restrictions.
-  - `summary` is a one-line description of the charm.
-  - `description` is a long-form description of the charm and its features.
-  It will also appear in the juju GUI.
-  - `tags` is a descriptive tag that is used to sort the charm in the store.
+This page covers the various fields that can be included. Some are required and
+many are optional.
 
+## Required fields
 
+Every charm should have these fields declared:
 
-Here's a valid metadata file:
+  - `name`: The charm name, which is used to form the charm URL.
+    - The following criteria are applied:
+          - Contains only characters `a-z`, `0-9`, and `-`
+          - Must start with `a-z`
+          - Must not end with a `-`
+          - May only end with digits if the digits are _not_ directly preceded by a space.
```

Comment:
to me, those hyphens are little difficult to spot at the end of the line. Maybe put `(hyphen)` or similar after them?

Reply:
:+1: